pages/portfolio3.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" type="text/css" href="../styles/style-portfolio.css">
+  <title>Portfolio</title>
+</head>
+
+<body>
+  <main class="grid-parent">
+    <header>
+      <h1 class="title">
+        Camilla Berretta, full-stack developer
+      </h1>
+    </header>
+    <section>
+    </section>
+
+    <article>
+      <h1>My Portfolio</h1>
+      <p>
+        C'est un trou de verdure où chante une rivière,
+        Accrochant follement aux herbes des haillons
+        D'argent ; où le soleil, de la montagne fière,
+        Luit : c'est un petit val qui mousse de rayons.
+
+        Un soldat jeune, bouche ouverte, tête nue,
+        Et la nuque baignant dans le frais cresson bleu,
+        Dort ; il est étendu dans l'herbe, sous la nue,
+        Pâle dans son lit vert où la lumière pleut.
+
+        Les pieds dans les glaïeuls, il dort. Souriant comme
+        Sourirait un enfant malade, il fait un somme :
+        Nature, berce-le chaudement : il a froid.
+
+        Les parfums ne font pas frissonner sa narine ;
+        Il dort dans le soleil, la main sur sa poitrine,
+        Tranquille. Il a deux trous rouges au côté droit.1
+
+        Octobre 1870
+      </p>
+
+      <nav class="box flex">
+        <p>
+          <a href="https://github.com/camilla122333/task-list-api">
+            <img src="../assets/annie-sprattpinkbackground.jpg" alt="project" class="center">
+            tasklist
+          </a>
+        </p>
+        <p>
+          <a href="https://github.com/camilla122333/viewing-party">
+            <img src="../assets/jason-leung-wmyE5IBiOmo-unsplash.jpg" alt="project" class="center">
+            viewing party
+          </a>
+        </p>
+        <p>
+          <a href="https://www.pasadenastringtheory.com/recordings">
+            <img src="../assets/annie-sprattpinkbackground.jpg" alt="project" class="center">
+            music projects
+         </a>
+        </p>
+      </nav>
+
+    </article>
```

Comment:
I think that the closing article tag should come directly after your block of text ends on line 44. Since an article element usually specifies independent, self-contained content.

Then you'd have your nav element with your projects as a sibling of article, instead of nested in article